README-WAF.md

```diff
@@ -62,7 +62,18 @@ waf list
 
 ## Program groups ##
 
-The programs in ardupilot are categorized into the following groups:
+Program groups are used to represent a class of programs. They can be used to
+build all programs of a certain class without having to specify each program. A
+program can belong to more than one group.
```

Comment:
I get that the use of _partition_ implies that, but if you hadn't mentioned it now, I would never think it that way.

Anyway, that sentence is in a different section - and it is after the confusing sentence... When I reviewed this I read all text, and that didn't clarified it for me. It may be my problem, but I still think the first sentence is confusing.